### ISSUE TITLE
'admin' should be disabled

### ISSUE THREAD

**Issue by Gustry** [kartoza/docker-geosafe#96]
CMIIW but the admin/admin login is always enabled by default.
I think it would be better to not provide an admin login by default. It should either a prompt in the ansible script for the login/password. I don't know if you can do that from rancher catalog.

**Comment by lucernae** [comment]
Yes, this also have been in my mind for so long. The way I propose to solve it (for production instance):

- keep admin/admin login enabled by default
- the admin of course should change his password in the first place
- prevent paver sync from loading sample_admin.json fixtures (where admin/admin login described) so it was not going to reset.

IMO, it is still a good way to provide default password for admin for first install. We can put the password in rancher catalog, but because this is not an application settings (a user settings), I think we should not put it there.

**Comment by vasuse7en** [comment]
@lucernae should we use "data migrations" instead of fixtures?

**Comment by lucernae** [comment]
This fixtures is Geonode's convention to prepare initial data. We will just have to make sure that some fixtures won't run twice if db already populated.

**Comment by gubuntu** [comment]
@vasuse7n would you like to take this on?

**Comment by vasuse7en** [comment]
@gubuntu yes i would like too.

**Comment by gubuntu** [comment]
Hi @vasuse7n did you make any progress with this? Have you upgraded recently with our latest work?

**Comment by vasuse7en** [comment]
No @gubuntu. I didn't upgrade with your latest updates. I will try to update this week and look into this issue. Also if there exists a public channel for geosafe in Kartoza slack channel, could you please share me the link? just for easy communication.
